doc/releases/changelog-dev.md

```diff
@@ -200,14 +217,14 @@
   [(#3958)](https://github.com/PennyLaneAI/pennylane/pull/3958)
   [(#3983)](https://github.com/PennyLaneAI/pennylane/pull/3983)
 
-<h3>Deprecations</h3>
+<h3>Deprecations 👋</h3>
```

Comment:
If this stays empty, ideally add a simple sentence saying there's none. Or "(None.)"